ogre_mesh_viewer.py

```diff
@@ -253,7 +252,8 @@ def preRenderTargetUpdate(self, evt):
 
         self.logwin.draw()
 
-        if self.app.attach_node is not None:
+        #if self.app.attach_node is not None:
+        if entity == None:
```

Comment:
https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/singleton-comparison.html

also do not leave commented code behind